Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/InfiniteCanvas.cs

```diff
@@ -348,8 +351,41 @@ public void ImportFromJson(string json)
         }
 
         /// <summary>
-        /// This event triggered after each render happended because of any change in the canvas elements.
+        /// This method exports the max possible view of the InfiniteCanvas drawings as offScreen drawings that can be converted to image.
+        /// Max is calculated using CanvasDevice.MaximumBitmapSizeInPixels
+        /// </summary>
+        /// <param name="stream">The target stream.</param>
+        /// <param name="bitmapFileFormat">the specified format.</param>
+        /// <returns>Task</returns>
+        public async Task SaveMaxViewAsync(IRandomAccessStream stream, BitmapFileFormat bitmapFileFormat)
```

Comment:
I recommend changing this method to `SaveBitmapAsync` to make it more descriptive. 
```suggestion
        public async Task SaveBitmapAsync(IRandomAccessStream stream, BitmapFileFormat bitmapFileFormat)
```

Reply:
Done